apps/fyle/actions.py

```
@@ -213,7 +213,7 @@ def update_failed_expenses(failed_expenses: List[Expense], is_mapping_error: boo
 
         # Update if token expired regardless of current state
         # Otherwise, skip dummy updates (if it is already in error state with the same error type)
-        if is_token_expired or not (expense.accounting_export_summary.get('state') == 'ERROR' and \
+        if is_token_expired or not (expense.accounting_export_summary.get('state') in ['ERROR', 'DELETED'] and \
             expense.accounting_export_summary.get('error_type') == error_type):
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)